dev/docker/hive/Dockerfile

```diff
@@ -139,9 +147,15 @@ ADD mapred-site.xml ${HADOOP_CONF_DIR}/mapred-site.xml
 ADD check-status.sh /tmp/check-status.sh
 
 ################################################################################
-# install hive
-ADD packages/apache-hive-${HIVE_VERSION}-bin.tar.gz /opt/
-RUN ln -s /opt/apache-hive-${HIVE_VERSION}-bin ${HIVE_HOME}
+RUN mkdir -p ${HIVE_HOME}/lib
```

Comment:
I think we didn't need to prepare to create `${HIVE_HOME}/lib` directory.

We only move `RUN ln -s /opt/mysql-connector-java-${MYSQL_JDBC_DRIVER_VERSION}/* ${HIVE_HOME}/lib` to `startup.sh`

Reply:
done.